release/data-release.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script is intended to be run from within the docker
+# container and to run the ETL pipeline with given configuration.
+
+# Configuration file to use should be set as ENV variable PUDL_SETTINGS
+# and defaults to /pudl/src/release/data-release.sh
+
+set -e
+
+# TODO(rousik): uhm, running within conda is fragile, maybe we can work around
+# this somehow?
+
+conda init bash
+eval "$(conda shell.bash hook)"
+source activate pudl-dev
+
+pudl_setup --pudl_in /pudl/inputs --pudl_out /pudl/outputs
+ferc1_to_sqlite --clobber --sandbox $PUDL_SETTINGS
+pudl_etl --clobber --sandbox $PUDL_SETTINGS
```

Comment:
I was using `--sandbox` because at a time it was the only working configuration. I see that production zenodo has been added since so we can easily drop this now.

Reply:
I'm thinking that this script is a temporary thing only. Once `ferc1_to_sqlite` is integrated into the ETL we might also think about running `pudl_etl` in a setup-less mode, e.g. by specifying `--pudl_in` and `--pudl_out` parameters and preparing the directory structure as a first step.

In the end, I think that `pudl_etl` should basically run everything, from preparing files and local storage all the way to uploading the artifacts to some cloud-based location.